packages/client/components/ActivityLibrary/CreateNewActivity/CreateNewActivity.tsx

```diff
@@ -0,0 +1,263 @@
+        <h1 className='text-lg font-normal'>
+          Choose an <span className='font-semibold'>Activity Format:</span>
+        </h1>
+        <RadioGroup.Root
+          className='mx-auto flex flex-col gap-8 sm:flex-row'
+          aria-label='Choose an Activity Format'
+          value={selectedActivity?.type}
+          onValueChange={handleActivitySelection}
+        >
+          {SUPPORTED_CUSTOM_ACTIVITIES.map((category) => {
+            return (
+              <RadioGroup.Item
+                key={category.title}
+                className='disabled:user-select-none group flex cursor-pointer flex-col items-start space-y-3 bg-transparent disabled:cursor-not-allowed disabled:opacity-50'
+                value={category.type}
+                disabled={!category.isEnabled}
+              >
+                <ActivityCard
+                  className='w-80 group-data-[state=checked]:ring-4 group-data-[state=checked]:ring-sky-500 group-data-[state=checked]:ring-offset-4'
+                  title={category.title}
+                  titleAs={CategoryTitle}
+                  imageSrc={category.image}
+                />
+                <div className='flex gap-x-3 p-3'>
+                  {category.badges.map((badge) => (
+                    <ActivityBadge key={badge.category} className={badge.theme}>
+                      {badge.title}
+                    </ActivityBadge>
+                  ))}
+                </div>
+                <div className='mx-5 space-y-2 text-left'>
+                  {category.phases.map((phase) => (
+                    <div key={phase.title}>
+                      <span className='font-semibold text-slate-800'>{phase.title}</span>{' '}
+                      {phase.description}
+                    </div>
+                  ))}
+                </div>
+              </RadioGroup.Item>
+            )
+          })}
+        </RadioGroup.Root>
+        <div className='flex flex-col items-center space-y-4 px-4'>
+          <div className='text-lg'>
+            <span className='font-semibold'>Select the team</span> to manage this new activity
+            template:
+          </div>
+          <div className='w-full px-4'>
+            <NewMeetingTeamPicker
+              teamsRef={data.viewer.teams}
+              selectedTeamRef={selectedTeam}
+              onSelectTeam={(teamId) => {
+                const newTeam = data.viewer.teams.find((team) => team.id === teamId)
+                newTeam && setSelectedTeam(newTeam)
+              }}
+            />
+          </div>
+        </div>
+        {error && <div className='px-4 text-tomato-500'>{error.message}</div>}
+        <div className='mt-auto flex w-full bg-slate-200 p-2 shadow-card-1'>
+          <PrimaryButton
+            className='mx-auto h-12 text-lg font-semibold'
+            onClick={createCustomActivityLookup[selectedActivity.type]}
+          >
+            Create template
```

Comment:
@enriquesanchez is the copy `Looks good!` from https://www.figma.com/file/PmE9A2ryODVzFPjZFJn7qi/Parabol-Activity-Library?node-id=915-28182&t=W7KL7GOIq7FnybxG-0 correct?

Reply:
@BartoszJarocki 🤔 @ackernaut do you want to keep "Looks good"?

@ackernaut I wonder if we should use a blue button instead (like "Done" when you edit a template), and only use the bright gradient for "Start activity"?

Reply:
I’m okay moving to the blue secondary button style instead of the primary gradient. If we want to _button up_ the label copy we could say ‘Create Activity’ or ‘Confirm Type & Team’ instead of ‘Looks Good!’

Reply:
Here’s a [view in Figma](https://www.figma.com/file/PmE9A2ryODVzFPjZFJn7qi/Parabol-Activity-Library?type=design&node-id=915%3A28182&t=Ld0uowolL1MzRkzU-1)

Reply:
@BartoszJarocki does that get you what you need to move forward?

Reply:
yeah, thank you @enriquesanchez @ackernaut 🙏